x-pack/legacy/plugins/lens/public/datatable_visualization_plugin/expression.tsx

```diff
@@ -113,8 +116,20 @@ export const getDatatableRenderer = (
   help: '',
   validate: () => {},
   reuseDomNode: true,
-  render: async (domNode: Element, config: DatatableProps, _handlers: unknown) => {
-    ReactDOM.render(<DatatableComponent {...config} formatFactory={formatFactory} />, domNode);
+  render: async (
+    domNode: Element,
+    config: DatatableProps,
+    handlers: IInterpreterRenderHandlers
+  ) => {
+    ReactDOM.render(
+      <DatatableComponent {...config} formatFactory={formatFactory} />,
+      domNode,
+      () => {
+        if (handlers.done) {
```

Comment:
It seems to me that `handlers.done` should always be there, and just be a noop if no handlers exist or whatever. Then consumers can just use it.

```js
ReactDOM.render(<Etc />, domNode, handlers.done);
```

Reply:
handlers.done should always be present